spec/features/comments_feature_spec.rb

```diff
@@ -0,0 +1,62 @@
+require 'rails_helper'
+
+feature "comments" do
+  context 'when not logged in' do
+    scenario 'a visitor cannot leave a comment' do
+      Picture.create(name:'test',
+                     avatar_file_name: 'test.jpg',
+                     avatar_file_size: '10',
+                     avatar_content_type: 'image/jpeg')
+      visit '/pictures'
+      click_link 'comment'
+      expect(page).to have_content 'Log in'
+    end
+  end
+
+  context 'when logged in' do
+    before do
+      sign_up
```

Comment:
Gnarly helpers. Who am I signing up as? What is this new_picture? What comment am I leaving?